tests/network/migration/test_masquerade_connectivity_after_migration.py

```diff
@@ -14,12 +14,36 @@
     fedora_vm_body,
     migrate_vm_and_verify,
     vm_console_run_commands,
-    wait_for_console,
 )
 
 LOGGER = logging.getLogger(__name__)
 
 
+def ping_via_console(src_vm, dst_vm):
+    """
+    Ping the destination VM from the source VM via the VM console.
+    Executes `ping` with `-c 10 -w 10` so the command succeeds only if all
```

Comment:
please see the requested docstring format in https://google.github.io/styleguide/pyguide.html#383-functions-and-methods